modules/carto/src/api/fetch-map.ts

```diff
@@ -271,16 +255,15 @@ export type FetchMapResult = ParseMapResult & {
 /* eslint-disable max-statements */
 export async function fetchMap({
   accessToken,
-  apiBaseUrl = SOURCE_DEFAULTS.apiBaseUrl,
+  apiBaseUrl = DEFAULT_API_BASE_URL,
   cartoMapId,
-  clientId = SOURCE_DEFAULTS.clientId,
-  headers = {},
+  clientId,
+  headers,
```

Comment:
Is this a change of behavior?

Reply:
Not "intended" to be a change of behavior (the defaults should be populated downstream, once, not in multiple places like they were here) but I do want to do a bit more testing to confirm this before merging. 👍🏻

Reply:
Tested on `test/apps/carto-map`, no changes found compared to 9.0-release. There is one failing map... 

```
b8abc46c-3c7f-489f-b16f-0664872ad82a
``` 

... but it also fails on 9.0.